cross/redis/patches/003-Makefile.patch

```diff
@@ -0,0 +1,11 @@
+--- src/Makefile-orig	2019-12-08 19:00:29.096004000 +0100
++++ src/Makefile	2019-12-08 19:00:39.076004000 +0100
+@@ -29,7 +29,7 @@
+ WARN=-Wall -W -Wno-missing-field-initializers
+ OPT=$(OPTIMIZATION)
+ 
+-PREFIX?=/usr/local
++#PREFIX?=/usr/local
```

Comment:
you can omit this patch (remove the whole file 003-Makefile.patch).
The ?= operator makes the assignment only when the variable is not defined yet. The framework will set the PREFIX and the makefile will use the right one (and output goes to work...../install/usr/local/redis/bin).

Reply:
After removing this patch, indeed compiling is still OK. It was due to my first test to cross compile without spksrc, so I forgot to remove this.